docs/tutorials/default-environment.md

```diff
@@ -49,15 +49,18 @@ or you can add a single line to your shell's RC file:
 
 === "Bash"
 
-    Add the following line to the very end of your `.bashrc` file:
+    Depending on the context, Bash will load different startup files.
```

Comment:
nit: I think it might be clearer to just say depending on whether it's a login shell

Reply:
I think most people don't know what a login shell is, so I think it's ok.